addon/mixins/ajax-request.js

```diff
@@ -39,11 +39,24 @@ const {
 } = Ember;
 const JSONAPIContentType = 'application/vnd.api+json';
 
+// Header names are case-insensitive
+// https://www.w3.org/Protocols/rfc2616/rfc2616.html
+function retrieveHeader(headers, name) {
+  var found = null;
+  Object.keys(headers).forEach((key) => {
+    let value = headers[key];
+    if (value.toLowerCase() === name.toLowerCase()) {
```

Comment:
@sandstrom Actually, before I go ahead and do this myself, do you want to take a stab at fixing this?  We need to compare the _key_ to the _name_ parameters, and then return the corresponding value, not compare against the value.